content/influxdb/v2.1/security/tokens/delete-token.md

```diff
@@ -15,15 +15,37 @@ Delete API tokens from the InfluxDB user interface (UI) or the `influx` command
 Once deleted, all users and external integrations using the API token will no longer
 have access to your InfluxDB instance.
 
+- [Delete tokens in the InfluxDB UI](#delete-tokens-in-the-influxdb-ui)
+- [Delete a token using the influx CLI](#delete-a-token-using-the-influx-cli)
+- [Delete a token using the InfluxDB API](#delete-a-token-using-the-influxdb-api)
+
 ## Delete tokens in the InfluxDB UI
 
+{{% oss-only %}}
+
 1. In the navigation menu on the left, select **Data (Load Data)** > **Tokens**.
```

Comment:
We probably just need to find/replace in the 2.1 and cloud docs:

`**Data (Load Data)** > **Tokens**`--> `**Load Data** > **Tokens**`

```suggestion
1. In the navigation menu on the left, select **Load Data** > **API Tokens**.
```